swssconfig/swssconfig.cpp

```diff
@@ -0,0 +1,203 @@
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include "logger.h"
+#include <fstream>
+#include "dbconnector.h"
+#include "producertable.h"
+#include "json.hpp"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
+
+int db_port							= 6379;
+const char* const hostname 			= "localhost";
+const char* const op_name 			= "OP";
+const char* const op_set 			= "SET";
+const char* const op_del 			= "DEL";
+const char* const name_delimiter 	= ":";
```

Comment:
Can you clarify the comment please? Are you referring to set/del?
Changed set/del to use SET_COMMAND/DEL_COMMAND from common/scheme.h